### PR TITLE
simplify skip condition

### PR DESCRIPTION
**Pull request by xantares**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ source:
 
 build:
   number: 0
-  skip: true          # [win and not (py34 or py35)]
+  skip: true          # [win and py27]
```
Comment (isuruf) [inline recipe/meta.yaml:19]: This was added because we only wanted to build with `py34 or py35` and not with other versions like `py36` now.

Comment (xantares) [reply recipe/meta.yaml:19]: oh ok.
why does the py36 fail by the way ?

Comment (isuruf) [reply recipe/meta.yaml:19]: It doesn't. Since the VC version of py35 and py36 are the same, there's no need to build twice.

Comment (xantares) [reply recipe/meta.yaml:19]: ok

**Pull request closed without merging**